safety_shield/config/robot_parameters_schunk.yaml

```diff
@@ -55,4 +55,31 @@ enclosures: [
   0.07,
   0.00049324, 0.0033497, 0.024003,
   -0.00049264, 0.0031812, 0.09,
-  0.072549]
\ No newline at end of file
+  0.072549]
+
+# transformation_matrices_joints : [
```

Comment:
transformation_matrices_joints can be deleted, because was only used for debugging